projects/observability/src/shared/components/cartesian/d3/axis/cartesian-axis.ts

```diff
@@ -235,4 +240,4 @@ export class CartesianAxis<TData = {}> {
   }
 }
 
-type DefaultedAxisConfig = Axis & Omit<Required<Axis>, 'scale' | 'crosshair' | 'min' | 'max'>;
+type DefaultedAxisConfig = Axis & Omit<Required<Axis>, 'scale' | 'crosshair' | 'min' | 'max' | 'tickDaysDistance'>;
```

Comment:
This naming seems very specific to the use case and not what I would expect in a generic component.

Reply:
I will change it for a less specific naming/data